app/assets/javascripts/components/story/StoryPopover.js

```diff
@@ -65,9 +65,9 @@ const StoryPopover = ({ story, children }) => (
   >
     {
       ({ ref }) => (
-        <bold ref={ref}>
+        <div ref={ref}>
```

Comment:
Two questions here:

1 - Why was it necessary?
2 - It will make it lose the bold effect, is it intended?

Reply:
React was giving a warning that it was an unsupported tag. This wasn't necessary since this div is just the popover target.